ideascube/migrations/0009_add_a_system_user.py

```diff
@@ -5,6 +5,13 @@
 
 
 def add_user(apps, *args):
+    # The apps.get_model() method returns us a 'fake' model with a
+    # default manager and the 'save' method is not overwrited
+    # and the user is not indexed at creation.
+    # This is not a problem here as we have post-migrate hook who reindex
+    # everything at end of migration. Moreover, it is better has we do not
+    # have to care about if the 'transient' db is existing or not when doing
+    # this migration.
```

Comment:
> I'm curious, what made you feel this comment was necessary?

The time I spent to understand all this stuff and the pity I've got for the next developer who will try to understand it again.